pkg/server/types.go

```diff
@@ -61,6 +62,8 @@ type ImmuServer struct {
 	metricsServer       *http.Server
 	mux                 sync.Mutex
 	RootSigner          RootSigner
+	startedAt           time.Time
+	ReadyChan           chan bool
```

Comment:
I'm thinking ReadyChan could be implemented as a callback channel provided as an argument while constructing or starting the server instance, instead of having it as part of the struct. Also it could be of type `chan struct{}`so to maintain same type as quit channel